Changes.md

```diff
@@ -13,6 +13,10 @@ Features
   - Added a visual indication to lights in the viewport when they are muted. Muted lights use a dark gray outline instead of yellow.
   - Cycles : Muted lights are disabled in renders.
   - Arnold : Muted lights are disabled in renders.
+- HierarchyView : Added Inclusions and Exclusions columns to manage the display of subsets of the scene in the Viewer via a VisibleSet.
```

Comment:
It does feel more natural to describe the concepts with the Visible Set front and centre. I was trying to avoid adding too many new terms to the user-facing explanations, but as you point out talking around the actual core concept does end up a bit counter productive...

I've tried the Visible Set focused Changes in 8f84dc7 and tooltips in c03926c. May still want to take another run at them, but we can always revisit after this is merged.